distributed/cli/tests/test_dask_submit.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import print_function, division, absolute_import
+from distributed.utils_test import valid_python_script, popen, invalid_python_script
+
+def test_dask_submit_cli(valid_python_script, invalid_python_script):
+    with popen(['dask-scheduler', '--port', '8989']) as sched:
+            with popen(['dask-submit', '127.0.0.1:8788', str(valid_python_script)]) as submit:
```

Comment:
Some whitespace alignment issues here.  Looks like you're slightly over-indented here.